packages/loader/container-loader/src/containerContext.ts

```diff
@@ -85,7 +86,7 @@ export class ContainerContext implements IContainerContext {
 			batch: IBatchMessage[],
 			referenceSequenceNumber?: number,
 		) => number,
-		public readonly submitSignalFn: (content: any, targetClientId?: string) => void,
+		public readonly submitSignalFn: (content: ISignalEnvelope, targetClientId?: string) => void,
```

Comment:
And if you change container-definitions' IContainerContext
```TypeScript
	readonly submitSignalFn: (contents: any, targetClientId?: string) => void;
```
to have `contents: unknown` is the build still happy?
To make this change, container-definitions' IContainerContext.submitSignalFn contents argument type will need to be ISignalEnvelope. It is an `@alpha` interface; so we should be able to change it, but does Loop (or anyone) use it today?

Reply:
Build doesn't work with `contents: unknown`. I did try to change `contents: ISignalEnvelope` for IContainerContext.submitSignalFn but it doesn't like this since ISignalEnvelope is marked as `internal`.

Reply:
Okay. Then `unknown` is the best we can do without verifying there is no Loop use.
If there isn't any Loop use, then we could go to `ISignalEnvelope` making it `@alpha` but it probably isn't worth the cost.
With the `unknown` change to definitions, it could be worth using `unknown | ISignalEnvelope` in the places you have changed with a comment explaining that `unknown` should be removed once `@alpha` is removed from `IContainerContext`. Additionally note that any changes to `submitSignalFn content` should be checked internally by temporarily changing `IContainerContext` and removing all `unknown`s.